cluster-operator/src/main/java/io/strimzi/operator/cluster/Main.java

```diff
@@ -171,7 +171,7 @@ static CompositeFuture deployClusterOperatorVerticles(Vertx vertx, KubernetesCli
                             "ABCDEFGHIJKLMNOPQRSTUVWXYZ" +
                             "0123456789");
 
-            kafkaClusterOperations = new KafkaAssemblyOperator(vertx, pfa, certManager, passwordGenerator, resourceOperatorSupplier, config);
+            kafkaClusterOperations = new KafkaAssemblyOperator(vertx, pfa, certManager, passwordGenerator, resourceOperatorSupplier, config, java.time.Clock.systemUTC());
```

Comment:
Is the original date also always in UTC? We need to make sure that users using different timezone do not have any breaking changes from this. E.g. that renewal happens at 3pm instead of 1am etc.

Reply:
The existing [dateSupplier](https://github.com/strimzi/strimzi-kafka-operator/blob/main/cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaAssemblyOperator.java#L648-L650) method in KAO is creating a instance of the `Date` class, which is in UTC IIRC. So the dateSupplier behavior is ignoring any system timezone and everything is in UTC.

However, this definitely needs testing.